endpoints/signup.go

```diff
@@ -28,6 +28,15 @@ func (c *Client) Signup(req types.SignupRequest) (*types.SignupResponse, error)
 		return nil, err
 	}
 
+	if req.ConfirmationRedirectUrl != "" {
+		q := r.URL.Query()
+		q.Add("redirect_to", req.ConfirmationRedirectUrl)
+		r.URL.RawQuery = q.Encode()
+
+		// Set up a client that will not follow the redirect.
+		c.client = noRedirClient(c.client)
```

Comment:
I can see that the JS client appears to be forwarding a `redirect_to` parameter like this, but In my testing, I was unable to get the server to actually return a redirect here. Were you able to successfully make use of this redirect param?